tests/unit/commands/test_completions.py

```diff
@@ -0,0 +1,143 @@
+from argparse import Action
+from collections import namedtuple
+from typing import Tuple
+
+import pytest
+
+from borgmatic.commands.completion import (
```

Comment:
So the convention in other tests in borgmatic would be to do this instead:

```python
from borgmatic.commands import completion as module
```

And then wherever you need to refer to a unit under test, it would look like `module.dedent_strip_as_tuple` or `module.exact_options_completion`.

The intent of this convention, BTW, is that it makes it real easy to identify the unit under test when reading a test function.